src/docs/user/references/ScriptLanguageSupport.adoc

```diff
@@ -396,6 +396,50 @@ Or another script binding (for example, USERSPACE):
 my $USERSPACE= $ENV{"USERSPACE"};
 ----
 
+=== PHP
+
+The PHP script engine features the following behaviors:
+
+* Script bindings are translated as environment variables.
+* Workflow variables are un-modifiable except by using a post-script written in another language such as the `update_variables_from_file` script (see <<_bash_and_shell,Bash script language>> for further explanations).
+* PHP script engine is meant to return HTML contents and makes use of link:../user/ProActiveUserGuide.html#_assigning_metadata_to_task_result[result metadata] through the *content.type*="text/html" link:../user/ProActiveUserGuide.html#_generic_information[generic information]. Using the script engine without defining this generic information in the Task is not recommended.
+* The _result_ of the script engine execution is a byte array containing the html content produced by the `php` command. In case the `php` command returns a non zero value, the result of the script engine execution will be the command exit code and the associated workflow task will be in error.
+* The PHP script engine is not well adapted to link:../user/ProActiveUserGuide.html#_pre_post_clean[pre], link:../user/ProActiveUserGuide.html#_pre_post_clean[post] or link:../user/ProActiveUserGuide.html#_pre_post_clean[clean] script execution. Even though they can be executed as such, they will not produce any html output in these contexts.
+
+NOTE: the `php` returns a non-zero exit code only in case of syntax errors. In case runtime issues occur, php will generally output warnings and the php command will return a zero exit code, not triggering a workflow task failure. This can be modified by using php exception mechanism and calling `exit(error_code)` in case of serious failures.
+
+Inside PHP, you can access environment variables using the *getenv()* function.
+
+The aim of next examples is to clarify how script engine bindings can be accessed inside PHP scripts.
```

Comment:
The aim of **the** next examples

Reply:
I rephrased by using "Next examples clarify"